docs/.vuepress/config.js

```diff
@@ -23,34 +23,55 @@ module.exports = {
     feedbackWidget: {
       docsRepoIssue: 'apostrophecms/apostrophe-documentation'
     },
-    logo: '/images/logo-sm.png',
+    logo: '/images/apos-dark.png',
     nav: [
       {
-        text: 'Docs Home',
-        link: '/'
+        text: 'v3 Docs',
+        link: 'https://v3.docs.apostrophecms.org'
```

Comment:
Similar to my review on https://github.com/apostrophecms/a3-docs/pull/104/files I think this would be better as a dropdown that shows there are two versions available. Can we update this here as well?